core/arch/aarch64/codec.txt

```diff
@@ -124,6 +124,11 @@
 ??--------xxxxxxxxxxxxxxxxx-----  mem12      # gets size from 31:30
 ----------xxxxxxxxxxxxxxxxx-----  mem12q     # size is 16 bytes
 ----------xxxxxxxxxxxxxxxxx-----  prf12      # size is 0 bytes (prefetch variant of mem12)
+----------xx--------x-----------  vindex_H   # Index for vector with half elements (0-7)
```

Comment:
Though one may feel that these belong together, they are not correctly ordered by pattern.

Reply:
Reordered